wiki/es/es-Tips-Tricks-More.md

```diff
@@ -120,25 +121,23 @@ Para enviar estas señales utilizando systemd, crea los siguientes dos archivos
 Para encender o apagar la grabación (dependiendo del estado actual):
 
 ~~~
-[Unit]
-Description=Toggle recording state of Jamulus server
-Requisite=Jamulus-Server
+ [Unit]
+ Description=Toggle recording state of Jamulus server 
+ Requisite=Jamulus-Server
 
-[Service]
-Type=oneshot
-ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server
+ [Service]
+ Type=oneshot ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server 
 ~~~
 
 Para empezar una nueva grabación:
 
 ~~~
-[Unit]
-Description=Toggle recording state of Jamulus server
-Requisite=Jamulus-Server
+ [Unit]
+ Description=Start a new recording on Jamulus server 
```

Comment:
This is used for switching recording on and of. Toggle is the correct word here.

Reply:
If it's wrong it has to be changed in the original. This is merely a copy/paste from there.

Reply:
Strange. @gilgongo might know something about this?

Reply:
As far as I know there is a distinction between toggling recordings on and off using SIGUSR2 and starting a new one using SIGUSR1. The latter creates a new directory to save the file to, and you use the former to control whether it's saving or not. So the docs are correct. I guess if there's a confusion between "on/off" and "new" we could clarify that maybe (eg "To control whether the recording is running or not")?

Reply:
BTW once 3.8 is out I plan to create a "Server Administration Guide" that will clarify this kind of stuff as it's currently a bit scattered between separate files (same for "User Manual").